turbinia/workers/analysis/windows_acct.py

```diff
@@ -148,7 +154,57 @@ def _extract_windows_hashes(self, result, location):
           if passwdhash in IGNORE_CREDS:
             continue
           creds.append(line.strip())
-          hashnames[passwdhash] = username
+          if passwdhash in hashnames:
+            hashnames[passwdhash] = hashnames[passwdhash] + ", " + username
+          else:
+            hashnames[passwdhash] = username
+      os.remove(hash_file)
+    else:
+      raise TurbiniaException('Extracted hash file not found.')
```

Comment:
small nit: Can we add `TurbiniaException` to the docstrings for thsi method (and the `_extract_ad_hashes()` method)?

Reply:
Done